### ISSUE TITLE
Compiling (Mono): GodotSharpTools build fails

### ISSUE THREAD

**Issue by LinuxUserGD** [godotengine/godot#18127]
**Godot version:**
master

**OS/device including version:**
Antergos OS (Arch Linux)

**Steps to reproduce:**
```
yaourt -S msbuild-stable
scons p=x11 tools=yes module_mono_enabled=yes mono_glue=no
```

**Error Output:**
```Swift
scons: Reading SConscript files ...
Enabling ALSA
Enabling PulseAudio
Checking for C header file mntent.h... (cached) yes
scons: done reading SConscript files.
scons: Building targets ...
[  0%] mono_build_solution(["bin/GodotSharpTools.dll"], ["modules/mono/editor/GodotSharpTools/GodotSharpTools.sln"])
MSBuild path: /usr/bin/msbuild

Unhandled Exception:
System.TypeInitializationException: The type initializer for 'System.Console' threw an exception. ---> System.TypeInitializationException: The type initializer for 'System.ConsoleDriver' threw an exception. ---> System.Exception: Magic number is wrong: 542
  at System.TermInfoReader.ReadHeader (System.Byte[] buffer, System.Int32& position) [0x00028] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.TermInfoReader..ctor (System.String term, System.String filename) [0x0005f] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.TermInfoDriver..ctor (System.String term) [0x00055] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.ConsoleDriver.CreateTermInfoDriver (System.String term) [0x00000] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.ConsoleDriver..cctor () [0x0004d] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
   --- End of inner exception stack trace ---
  at System.Console.SetupStreams (System.Text.Encoding inputEncoding, System.Text.Encoding outputEncoding) [0x00007] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.Console..cctor () [0x0008e] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
   --- End of inner exception stack trace ---
  at Microsoft.Build.CommandLine.MSBuildApp.Execute (System.String commandLine) [0x003c3] in <abb2bae1109843268182b1c720ff6f2a>:0 
  at Microsoft.Build.CommandLine.MSBuildApp.Main () [0x00028] in <abb2bae1109843268182b1c720ff6f2a>:0 

Unhandled Exception:
System.TypeInitializationException: The type initializer for 'System.Console' threw an exception. ---> System.TypeInitializationException: The type initializer for 'System.ConsoleDriver' threw an exception. ---> System.Exception: Magic number is wrong: 542
  at System.TermInfoReader.ReadHeader (System.Byte[] buffer, System.Int32& position) [0x00028] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.TermInfoReader..ctor (System.String term, System.String filename) [0x0005f] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.TermInfoDriver..ctor (System.String term) [0x00055] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.ConsoleDriver.CreateTermInfoDriver (System.String term) [0x00000] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.ConsoleDriver..cctor () [0x0004d] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
   --- End of inner exception stack trace ---
  at System.Console.SetupStreams (System.Text.Encoding inputEncoding, System.Text.Encoding outputEncoding) [0x00007] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.Console..cctor () [0x0008e] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
   --- End of inner exception stack trace ---
  at Microsoft.Build.CommandLine.MSBuildApp.Execute (System.String commandLine) [0x003ed] in <abb2bae1109843268182b1c720ff6f2a>:0 
  at Microsoft.Build.CommandLine.MSBuildApp.Main () [0x00028] in <abb2bae1109843268182b1c720ff6f2a>:0 
[ERROR] FATAL UNHANDLED EXCEPTION: System.TypeInitializationException: The type initializer for 'System.Console' threw an exception. ---> System.TypeInitializationException: The type initializer for 'System.ConsoleDriver' threw an exception. ---> System.Exception: Magic number is wrong: 542
  at System.TermInfoReader.ReadHeader (System.Byte[] buffer, System.Int32& position) [0x00028] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.TermInfoReader..ctor (System.String term, System.String filename) [0x0005f] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.TermInfoDriver..ctor (System.String term) [0x00055] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.ConsoleDriver.CreateTermInfoDriver (System.String term) [0x00000] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.ConsoleDriver..cctor () [0x0004d] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
   --- End of inner exception stack trace ---
  at System.Console.SetupStreams (System.Text.Encoding inputEncoding, System.Text.Encoding outputEncoding) [0x00007] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
  at System.Console..cctor () [0x0008e] in <0cc37f4786fa491387f4cb0ad6d68b47>:0 
   --- End of inner exception stack trace ---
  at Microsoft.Build.CommandLine.MSBuildApp.Execute (System.String commandLine) [0x003ed] in <abb2bae1109843268182b1c720ff6f2a>:0 
  at Microsoft.Build.CommandLine.MSBuildApp.Main () [0x00028] in <abb2bae1109843268182b1c720ff6f2a>:0 
scons: *** [bin/GodotSharpTools.dll] RuntimeError : GodotSharpTools build failed
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/SCons/Action.py", line 1197, in execute
    result = self.execfunction(target=target, source=rsources, env=env)
  File "/run/media/user/user/git/godot/modules/mono/SCsub", line 198, in mono_build_solution
    raise RuntimeError('GodotSharpTools build failed')
RuntimeError: GodotSharpTools build failed
scons: building terminated because of errors.

```

**Comment by LinuxUserGD** [comment]
As described in https://wiki.archlinux.org/index.php/Mono, I've tried using xterm instead of nautilus and it works!

**Comment by neikeq** [comment]
The fix will be included in Mono 5.12.
